### PR TITLE
Fix hbs home page

### PR DESCRIPTION
**Pull request by aminalakhsham**
Relates #33 
fix button to have anchor to request in routes we need

### CODE REVIEW

**Comment by InassTubail** [comment]
@aminalakhsham  In another time waiting for approval :)

**Comment by aminalakhsham** [comment]
Sorry , i do that by mistake ...
plz add your review :)
@InassTubail

**Comment by InassTubail** [comment]
No problem, just try not to repeat that again.
Good job :+1:

**Comment by aminalakhsham** [comment]
> No problem, just try not to repeat that again.
> Good job

thank you :D

File: src/views/home.hbs [hunk 1]
```diff
@@ -14,8 +14,8 @@
       conduct hospital programs to improve organ donation rate in India.
     </p>
     <div class="main--header-request">
-      <div class="main--header-forYou">For You</div>
-      <div class="main--header-helpOthers">Help Others</div>
+      <button class="main--header-forYou"><a href="/request" class="main--header-forYouA">For You</a></button>
+      <button class="main--header-helpOthers"><a href="/help-other" class="main--header-helpOthersA">Help Others</a></button>
```
Comment (tomduggan85) [inline src/views/home.hbs:18]: @aminalakhsham Hi! I think you could change some of these class names to make your css a little easier to write later.  If the "For You" and "Help Others" buttons are intended to look the same, then you could use the same class name for both (for example `<button class="main--header-button"`).
Another thing you could do is remove the class on the `<a>` that's  inside each button, and just use a longer css selector when it's time to create the css.  If you took away the class on both of those anchor tags, you could still style them by using:
```
.main--header-forYou a,
.main--header-helpOthers a {
```
.. in your css (or `.main--header-button a` if you made the first change I suggested too)

I think these steps would enable you to write fewer css selectors when it comes time to write the css for this page (and so you'll have less duplication in your css)

**Pull request merged**